content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -406,8 +462,8 @@ The response would look the same as for non-dry-run request, but the values of s
 Authorization for dry-run and non-dry-run requests is identical. Thus, to make
 a dry-run request, the user must be authorized to make the non-dry-run request.
 
-For example, to run a dry-run `PATCH` for Deployments, you must have the
-`PATCH` permission for Deployments, as in the example of the RBAC rule below.
+For example, to run a dry-run Patch for Deployments, you must have the
```

Comment:
nit: "dry run" or "dry-run"
nit: Leave PATCH as `PATCH`

Reply:
I think `patch` is most appropriate here (matching the RBAC rule) but would need tech review / advice to confirm.

Reply:
Addressed.